prow/config.yaml

```diff
@@ -160,6 +160,10 @@ branch-protection:
           restrictions: # only allow admins
             users: []
             teams: []
+        publishing-bot:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
```

Comment:
Can we migrate this test to prow? Should be a trivial change (aka use a bash script instead of the travis.yaml file)

Reply:
In order to migrate you'll need two things:
1) an image that has all the dependencies required to run your test (minus whatever deps the test itself installs)
2) encapsulate the test inside a single command + args (bash script, whatever).

Here's [an example](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/org/kubernetes-org-jobs.yaml#L3-L13) job that does this: this particular test depends on a) the source code and b) bazel. So we give it a bazel image and the command `bazel test //...`.

The second job for that repo encapsulates a slightly more complicated set of commands into a `hack/verify-all.sh` script, which becomes the command.

Reply:
(doing this automatically integrate with the rest of our tooling like testgrid, etc.